lib/core/test/unit/Cardano/Wallet/Primitive/AddressDerivation/IcarusSpec.hs

```diff
@@ -69,24 +70,24 @@ import qualified Data.Text as T
 spec :: Spec
 spec = do
     describe "Golden Tests - Icarus' style addresses" $ do
-        let seed0 = Passphrase "4\175\242L\184\243\191 \169]\171 \207\r\v\233\NUL~&\ETB"
-
-        goldenAddressGeneration $ GoldenAddressGeneration
-            seed0 (toEnum 0x80000000) UTxOExternal (toEnum 0x00000000)
-            "Ae2tdPwUPEZGQVrA6qKreDzdtYxcWMMrpTFYCpFcuJfhJBEfoeiuW4MtaXZ"
-
-        goldenAddressGeneration $ GoldenAddressGeneration
-            seed0 (toEnum 0x80000000) UTxOExternal (toEnum 0x0000000E)
-            "Ae2tdPwUPEZDLWQQEBR1UW7HeXJVaqUnuw8DUFu52TDWCJbxbkCyQYyxckP"
-
-        goldenAddressGeneration $ GoldenAddressGeneration
-            seed0 (toEnum 0x8000000E) UTxOInternal (toEnum 0x0000002A)
-            "Ae2tdPwUPEZFRbyhz3cpfC2CumGzNkFBN2L42rcUc2yjQpEkxDbkPodpMAi"
-
+--        let seed0 = Passphrase "4\175\242L\184\243\191 \169]\171 \207\r\v\233\NUL~&\ETB"
+--
+--        goldenAddressGeneration $ GoldenAddressGeneration
```

Comment:
We cannot use the seed (which is really entropy) anymore.

We could
1. Remove the golden and not replace it
2. Replace it with another golden (from a mnemonic)

Reply:
I'm just removing it.